modules/ROOT/pages/dw-core-functions-filterobject.adoc

```diff
@@ -95,9 +95,38 @@ output application/json
 { "1": "a" }
 ----
 
+=== Example
+
+This example outputs an object with only values other than `null` from another JSON object.
```

Comment:
This example outputs an object **that contains** only **the** values that are not `null` **in the input** JSON object.